cmd/oci-runtime-tool/generate.go

```diff
@@ -608,6 +613,18 @@ func parseRlimit(rlimit string) (string, uint64, uint64, error) {
 	return parts[0], uint64(hard), uint64(soft), nil
 }
 
+func parseNamespace(ns string) (string, string, error) {
+	parts := strings.SplitN(ns, ":", 2)
```

Comment:
You can validate it against the spec's whitelist.  But I'm fine punting validation to a post-generate call (for callers who care).  Maybe they intend to fix the values in post-processing.